### PR TITLE
Annotation Pipeline (manual workflow fixes)

### PR DESCRIPTION
**Pull request by bonjarlow**
## Addresses
#96

This PR has tried and true fixes in the manual workflow merged last time. Primarily cleans up path issues that arose from running populate_labelstudio.py in a different location. This also simplifies the output folders for batch_info, cache, and tag collected csv's.

## Description

`annotation_pipeline/populate_labelstudio.py`
- path clean up. regardless of manual workflow or running locally, populate_labelstudio.py is now designed to be run from the parent directory (data-source-identification/)

`annotation_pipeline/config.ini`
- new config file for populate_labelstudio to specify save location in annotation_pipeline/data

`annotation_pipeline/README.md`
- updated instructions for running populate_labelstudio locally

`.github/workflows/populate_labelstudio.yml`
- fixes to yaml (string to bool (oops))
- pushes output logs to annotation_pipeline/data

### CODE REVIEW

**Review by josh-chamberlain — APPROVED** [review]
Thanks for the quick changes!

**Comment by maxachis** [comment]
Merged! I especially liked the thorough comments and type hinting -- that makes the overall logic much easier for me to follow. I do have a few nits in that I felt that some of the logic could benefit from some refactoring (mainly just extracting some functions), but I didn't feel that was sufficient to block the merge (and also I should have mentioned them in the larger review). Nice work, Jon!

**Comment by josh-chamberlain** [comment]
@bonjarlow @maxachis so close: https://github.com/Police-Data-Accessibility-Project/data-source-identification/actions/runs/10165804448

**Comment by bonjarlow** [comment]
@josh-chamberlain @maxachis here's the error I found in the main script execution:

Cannot access content at: https://huggingface.co/api/datasets/PDAP/unlabeled-urls/commit/main.
If you are trying to create or update content,make sure you have a token with the `write` role.

For the hugging face token used in github secrets, can you double check that it has write permissions? I see that you're the admin, but as far as I can tell I don't see write permission stated explicitly like it is for other members.

**Comment by josh-chamberlain** [comment]
@bonjarlow my fault! the token was given read-write access...to my **user**.

**Comment by bonjarlow** [comment]
@josh-chamberlain no worries! thanks for debugging with me :grin:

**Comment by bonjarlow** [comment]
@josh-chamberlain hmm the former run looks like it was a new issue with common crawl server being temporarily unavailable. that's a new one, and there's a typo in the exceptions block I'll fix right now.

the most recent run worked! until it didn't. it looks like the bot is not allowed to push to this branch without a PR first.

remote: error: GH006: Protected branch update failed for refs/heads/main.        
remote: error: Changes must be made through a pull request.

Is there a way to give the bot permission to write these files without a PR each time? or would you not mind it needing to be verified before each merge

**Comment by josh-chamberlain** [comment]
@bonjarlow Yeah, I retried with a different CC version and the first issue went away! Thanks for fixing the typo, though.

I wouldn't mind if it just created a PR for someone to approve. It doesn't seem easy to allow github actions to bypass the branch protection rules.

**Comment by bonjarlow** [comment]
@josh-chamberlain roger, I'll get that set up! third times the charm

**Pull request merged**
